boots/fields/tests/tests.py

```diff
@@ -50,9 +46,21 @@ def test_dropdown_widget(self):
         needle = '<li><a href="http://fark.com">Fark Thing</a></li>'
         self.assertInHTML(needle, result)
 
+    def test_radio_widget(self):
+        choices = [{'beatles': 'John', 'label': 'Beatle1'},
+                   {'beatles': 'Paul', 'label': 'Beatle2'},
+                   {'beatles': 'George', 'label': 'Beatle3'},
+                   {'beatles': 'Ringo', 'label': 'Beatle4'}]
+        radio_widget = widgets.RadioWidget(radio={'choices': choices},
+                                           attrs={'id': 'testid'})
+
+        result = radio_widget.render('fake_name', 'fake_value', attrs=radio_widget.attrs)
+        print result
+        needle = '<li>Ringo</li>'
```

Comment:
This test is failing because of 
<li><input type="radio" , aria-label="Favourite Beatle">&nbsp;&nbsp;{{ choice.beatles }}</li>

Notice the difference? The input tag and the two non-breaking spaces?